package.json

```diff
@@ -74,7 +74,7 @@
     "@angular/platform-browser-dynamic": "^11.2.14",
     "@angular/router": "^11.2.14",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "5.0.50-sscs-sit-release",
+    "@hmcts/ccd-case-ui-toolkit": "5.0.49-restricted-case-access-v2",
```

Comment:
Same concern about version number going backwards

Reply:
I have updated the version.